### PR TITLE
[IMP] pos: remove vantiv integration

### PR DESCRIPTION
**Pull request by dikd-odoo**
Removed Vantiv integration from 17.5 Sales POS documentation:

    Deleted Vantiv page from Point of Sale > Payment Methods > Terminals
    Updated Payment Terminals page content and image

Previously peer reviewed by JERO - PR #10426 

-cancelled PRS:
#10429 
#10426

Target: 17.5+

[Task](https://www.odoo.com/odoo/my-tasks/4077781?cids=3)

### CODE REVIEW

**Comment by dikd-odoo** [comment]
Hi @samueljlieber this PR is ready for technical review, when you have a chance! This is the revised master PR from #10429 and #10426 to remove the Vantiv POS terminal from 17.5+. Thank you again for your help on this!

File: content/applications/sales/point_of_sale/payment_methods/terminals/payment-terminals.png
Comment (samueljlieber) [inline content/applications/sales/point_of_sale/payment_methods/terminals/payment-terminals.png:1]: Please reduce this image width to be at or below 933px, as well as please compress this image with pngquant :)

File: redirects/17.0.txt [hunk 1]
```diff
@@ -96,6 +96,7 @@ applications/marketing/sms_marketing/pricing/pricing_and_faq.rst applications/ma
 applications/sales/crm/acquire_leads/generate_leads.rst applications/sales/crm/acquire_leads/email_manual.rst                                             #generate_leads --> email_manual
 applications/sales/crm/pipeline/multi_sales_team.rst applications/sales/crm/pipeline/manage_sales_teams.rst                                               # multi_sales_team --> manage_sales_teams
 applications/sales/point_of_sale/pricing/digital_labels.rst applications/sales/point_of_sale/pricing/electronic_labels.rst                                # digital_labels --> electronic_labels
+applications/sales/point_of_sale/payment_methods/terminals/vantiv.rst applications/sales/point_of_sale/payment_methods/terminals.rst		#remove vantiv integration
```
Comment (samueljlieber) [inline redirects/17.0.txt:99]: Nitpick for the redirect comment :) I suggest we stick with the existing convention of describing the redirect using arrows
```suggestion
applications/sales/point_of_sale/payment_methods/terminals/vantiv.rst applications/sales/point_of_sale/payment_methods/terminals.rst		# terminals/vantiv --> terminals
```

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @dikd-odoo! 
Thank you for re-opening this PR on master! I have just a couple quick technical fixes, and we should be good to go!

Please correct the following and tag me once more for another quick look. Thank you!

Please update the commit message to follow [the convention](https://github.com/odoo/documentation/blob/master/commit_template.txt) (missing the app/module)
Can use: `git commit --amend`
```
[IMP] pos: remove vantiv integration
```

Thank you for your work!

**Comment by dikd-odoo** [comment]
Hi @samueljlieber thank you so much for your review and also helping me with merge conflicts! I implemented your changes, this is ready for another look when you have a chance. Thanks again!

**Pull request closed without merging**